kingpin/actors/aws/elb.py

```diff
@@ -228,3 +216,93 @@ def _execute(self):
             yield utils.tornado_sleep(3)
 
         raise gen.Return()
+
+
+class UseCert(ELBBaseActor):
+
+    """Find a server cert in IAM and use it for a specified ELB."""
+
+    all_options = {
+        'name': (str, REQUIRED, 'Name of the ELB'),
+        'region': (str, REQUIRED, 'AWS region name, like us-west-2'),
+        'cert_name': (str, REQUIRED, 'Unique IAM certificate name, or ARN'),
+    }
+
+    # Local copy of {cert_name: cert_arn} dictionary
+    cert_arn_cache = {}
```

Comment:
Oh no no, this is super helpful. It's not re-using Actor object, it's re-using Actor class.
Updating 8 ELBs I create 8 Actors, all of which fetch the ARN based on the cert name.